lib/internal/modules/esm/hooks.js

```diff
@@ -162,12 +161,9 @@ class Hooks {
     } = pluckHooks(exports);
 
     if (globalPreload && !initialize) {
-      if (globalPreloadWarned === false) {
-        globalPreloadWarned = true;
-        emitExperimentalWarning(
-          '`globalPreload` will be removed in a future version. Please use `initialize` instead.',
-        );
-      }
+      emitExperimentalWarning(
+        'It is recommended to use `initialize` instead of `globalPreload`, which',
```

Comment:
```suggestion
        '`globalPreload` will be removed in a future version. Please use `initialize` instead. `globalPreload`',
```

To cause:

```
(node:9201) ExperimentalWarning: `globalPreload` will be removed in a future version. Please use `initialize` instead. `globalPreload` is an experimental feature and might change at any time
```

Reply:
I feel that the `might change at any time` is a bit redundant with the `will be removed in a future version` part.

Reply:
Yes, but unless you want to refactor this to not use `emitExperimentalWarning` then I think we’re stuck with it.

It’s also not _wrong;_ `globalPreload` can change any time before it eventually gets removed. As in, not only should you not be relying on `globalPreload` because it’s going away, but _also_ it might have breaking changes between now and its eventual removal. Beware!

Reply:
The problem I have with the suggested wording is that we don't know and can't know if `globalPreload` will be indeed removed. It's the plan, but the plan can change if new things come up. I think we should land something to fix the warning ASAP so it's part of the next release, the current warning looks unprofessional.
```suggestion
        '`globalPreload` is planned for removal in favor of `initialize`. `globalPreload`',
```

If you don't like it, feel free to takeover this PR and let's land this already.